torch/_torch_docs.py

```diff
@@ -1788,6 +1788,35 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.clamp`.
 """.format(**common_args))
 
+add_docstr(torch.column_stack,
+           r"""
+column_stack(tensors, *, out=None) -> Tensor
+
+Stack 1D tensors as columns into a 2D tensor, 2D tensors are stacked just like :func:`torch.hstack`.
+
+Args:
+    tensors (sequence of Tensors)
+
+Keyword args:
+    {out}
+
+Example::
+
+    >>> a = torch.tensor([1, 2, 3])
+    >>> b = torch.tensor([4, 5, 6])
+    >>> torch.column_stack((a,b))
+    tensor([[1, 4],
+        [2, 5],
+        [3, 6]])
+    >>> a = torch.tensor([[1],[2],[3]])
```

Comment:
I get what this example is trying to do but I think it could more distinct from the previous example by creating two tensors (using arange, for example), then reshaping them to be 2D, then column_stacking them plus a 1D tensor or two. That would highlight that 1D and 2D tensors can both appear in tensors.

Reply:
Thanks for pointing this out.
The latest example manipulates `torch.arange` to generate the 1D tensor, `torch.arange` with `torch.reshape` for the 2D tensor, and construct a tuple with one 1D tensor and two 2D tensors as the input of `torch.column_stack`.